fluentlenium-assertj/src/test/java/org/fluentlenium/integration/localtest/LocalFluentCase.java

```diff
@@ -29,7 +29,7 @@ public WebDriver getDefaultDriver() {
     }
 
     public static String getPath() {
-        String currentDir = System.getProperty("user.dir");
+        String currentDir = System.getProperty("user.dir").replace('\\', '/');
```

Comment:
I do not understand why are you doing such replacement? Java has mechanism to recognize what OS is running on and what is the path separator.